cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildCommandTest.java

```diff
@@ -756,4 +756,32 @@ public void testCustomTargetDirWithTests() {
                 ".json")));
     }
 
+    @Test(description = "Build an empty library module")
+    public void testBuildEmptyLibModule() throws IOException {
```

Comment:
Shall we rename the test to `testBuildEmptyPackage`? We do not need to specify that it is a `lib` We assume that build command will be run on applications only

Reply:
Renamed to `testBuildEmptyPackage`